gitops/snapshot.go

```diff
@@ -122,6 +128,26 @@ func SetSnapshotIntegrationStatusAsInvalid(snapshot *applicationapiv1alpha1.Snap
 	})
 }
 
+// SetSnapshotIntegrationStatusAsInProgress sets the HACBS integration status condition for the Snapshot to In Progress.
+func SetSnapshotIntegrationStatusAsInProgress(snapshot *applicationapiv1alpha1.Snapshot, message string) {
+	meta.SetStatusCondition(&snapshot.Status.Conditions, metav1.Condition{
+		Type:    HACBSIntegrationStatusCondition,
+		Status:  metav1.ConditionFalse,
```

Comment:
In light of the [guidance in the meta help](https://github.com/redhat-appstudio/integration-service/pull/84/files#r1060320035), maybe setting this to `metav1.ConditionUnknown` would make more sense?

Reply:
Thanks for the update here!